Builder workflow files

vv-monsalve opened the pull request:
Hi @mightybart, this PR includes the files needed for using Builder:

- [x] Source files include a new set of instances added to create the `fvar` info in the VF
- [x] Style-linking Regular-Bold fixes
- [x] Config.yml file in the `glyphs-decomposed` folder
- [x] `buildB.sh` file, that basically triggers the Builder and deletes the unnecessary static fonts created
- [x] New folder structure in the `fonts` directory

Builder wraps-up our tools and perform all the basic production-postproduction fixes. With this workflow, you wouldn't need the additional .ttx, sh or py file to create the fonts.

I've created a new `Builder-Montagu` branch so you can merge the PR there first to get familiar with the workflow.

Please merge it and let me know if you have any questions.

vv-monsalve commented:
Hi @mightybart, let me know if you have any question

mightybart commented:
Tested, works great. I would clean up the "sources" folder and delete the previous build.sh, .ttx & .py. We still need .designspace file right?

I had to manually "merge" the glyphs source, I made some design improvements in between. I assume the extra instances are there to fix some table issues? Medium had optical size 100 (should be 144), so I fixed this little thing.

The only thing that does not seem right to me is the "fonts/webfonts" folder. It looks like there are some unnecessary styles, and woff2 variable font is missing. Or is this OK?

vv-monsalve commented:
> We still need .designspace file right?

The configuration is currently taking the glyphs file as a source. It was done to avoid having to maintain two different "sources", although Builder could take the designspace as the source file, you could use it if prefered.

> I assume the extra instances are there to fix some table issues?

Yes, the source file includes a new set of instances added to create the `fvar` info in the VF

> Medium had optical size 100 (should be 144), so I fixed this little thing

It was an error in that case, great that you caught and fixed it. Please inspect everything just in case.

> and woff2 variable font is missing. Or is this OK?

We are producing by default only ttf variable. As in this case we are still using a `.sh` file you could include that final step in it if you prefer to include them in the upstream (your) repository.
